alt-tab-macos/ui/Application.swift

```diff
@@ -61,11 +61,13 @@ class Application: NSApplication, NSApplicationDelegate, NSWindowDelegate {
     }
 
     @objc func showCenteredPreferencesPanel() {
-        showCenteredPanel(preferencesPanel!)
+        showCenteredFrontPanel(preferencesPanel!)
     }
 
-    func showCenteredPanel(_ panel: NSPanel) {
-        panel.center()
```

Comment:
btw. we can't use .center() any longer since that seems to set the screen to NSScreen.main